client/src/hoc/PinnedContent.scss

```diff
@@ -0,0 +1,15 @@
+@import "../common_css/common-variables";
+@import "../extended_bootstrap_css/bootstrap-wet-fixes-extensions.scss";
+
+.btn-pin {
+    @extend .btn-ib-primary;
+
+    &:hover {
+        svg > g {
+            path, rect {
+                fill: $textLightColor;
+                stroke: $textLightColor;
+            }
+        }
+    }
+}
```

Comment:
Look at `_IconWrapper` in `icons.js`. To get hover colours on any of the icon components, just need to pass in an `alternate_color` prop